app/controllers/data_explorer/query_controller.rb

```diff
@@ -65,7 +65,14 @@ def group_reports_show
 
     respond_to do |format|
       format.json do
-        render_serialized @query, DataExplorer::QuerySerializer, root: 'query'
+        query_group = DataExplorer::QueryGroup.find_by(query_id: @query.id, group_id: @group.id)
+
+        # TODO this has a messy return, a query in a query, and a query_group in a query_group. Fix.
+        render json: {
+          query: serialize_data(@query, DataExplorer::QuerySerializer),
```

Comment:
You just need to add `root: nil` to the end of each of these `serialize_data` calls, otherwise it uses the JSON key as the key for the serialized data, which causes the double up

Reply:
Done.